test/configuration_unittest.cc

```diff
@@ -18,6 +18,7 @@ void VerifyDefaultConfig(const Configuration& config) {
             "v1beta2/projects/{{project_id}}/resourceMetadata:batchUpdate",
             config.MetadataIngestionEndpointFormat());
   EXPECT_EQ(8*1024*1024, config.MetadataIngestionRequestSizeLimitBytes());
+  EXPECT_EQ(1000, config.MetadataIngestionRequestSizeLimitCount());
```

Comment:
Please also add a check in the PopulatedConfig test for the non-default case.

Reply:
done